### PR TITLE
DOCS-12645 clarify monitoring 

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/index-self-managed.adoc [hunk 1]
```diff
@@ -108,6 +108,14 @@ Runtime Fabric on Self-Managed Kubernetes does not include external log forwardi
 
 For Titanium customers, Runtime Fabric on Self-Managed Kubernetes supports logging using Anypoint Monitoring. See xref:monitoring::logs.adoc[Logs in Anypoint Monitoring] for more information.
 
+=== Monitoring
+
+xref:monitoring::index.adoc[Anypoint Monitoring] provides basic metrics for applications and API gateways deployed to Runtime Fabric.
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/index-self-managed.adoc]: `basic` jumps out at me here. Is there a reason for including it?

File: modules/ROOT/pages/index-self-managed.adoc [hunk 2]
```diff
@@ -108,6 +108,14 @@ Runtime Fabric on Self-Managed Kubernetes does not include external log forwardi
 
 For Titanium customers, Runtime Fabric on Self-Managed Kubernetes supports logging using Anypoint Monitoring. See xref:monitoring::logs.adoc[Logs in Anypoint Monitoring] for more information.
 
+=== Monitoring
+
+xref:monitoring::index.adoc[Anypoint Monitoring] provides basic metrics for applications and API gateways deployed to Runtime Fabric. 
+
+To collect metrics, Anypoint Monitoring sidecars run in all pods deployed to Runtime Fabric. See xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric] for details, including how to enable or disable monitoring.
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/index-self-managed.adoc]: pods -> Applications

File: modules/ROOT/pages/index-self-managed.adoc (hunk 1 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/index-self-managed.adoc]: I took that from existing language in the docs, but I agree it's not needed. I'll cut it

File: modules/ROOT/pages/index-self-managed.adoc (hunk 2 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/index-self-managed.adoc]: got it

File: modules/ROOT/pages/index-self-managed.adoc (hunk 1 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/index-self-managed.adoc]:
```suggestion
xref:monitoring::index.adoc[Anypoint Monitoring] provides metrics for applications and API gateways deployed to Runtime Fabric. 
```

File: modules/ROOT/pages/index-self-managed.adoc [hunk 3]
```diff
@@ -108,6 +108,14 @@ Runtime Fabric on Self-Managed Kubernetes does not include external log forwardi
 
 For Titanium customers, Runtime Fabric on Self-Managed Kubernetes supports logging using Anypoint Monitoring. See xref:monitoring::logs.adoc[Logs in Anypoint Monitoring] for more information.
 
+=== Monitoring
+
+xref:monitoring::index.adoc[Anypoint Monitoring] provides basic metrics for applications and API gateways deployed to Runtime Fabric. 
+
+To collect metrics, Anypoint Monitoring sidecars run in all pods deployed to Runtime Fabric. See xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric] for details, including how to enable or disable monitoring. 
+
+Runtime Fabric does not support third-party monitoring solutions.
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/index-self-managed.adoc]: I don't think we need to say this. Some monitoring solutions can inject themselves into running pods, for example https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-container-platforms/kubernetes/deploy-oneagent-on-kubernetes-for-application-only-monitoring/ - and I don't think we could/should stop customers from doing that.

The truth is that we do not give customers the ability to _configure_ any specific monitoring solution from RTF itself. We do not have integration with specific vendors like Datadog or NewRelic etc. 

How about something softer eg  "does not *provide* support for integrating third-party monitoring solutions"?

@vli63127

File: modules/ROOT/pages/index-self-managed.adoc (hunk 2 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/index-self-managed.adoc]:
```suggestion
To collect metrics, Anypoint Monitoring sidecars run in all applications deployed to Runtime Fabric. See xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric] for details, including how to enable or disable monitoring. 
```

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 4]
```diff
@@ -3,15 +3,15 @@
 Applications and API gateways deployed to Runtime Fabric include native support for Anypoint Monitoring to allow viewing metrics within Anypoint Platform.
 
 [NOTE]
-Monitoring solutions other than Anypoint Monitoring are not supported.
+Runtime Fabric does not support monitoring solutions other than Anypoint Monitoring.
 
 == Anypoint Monitoring
 
-xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule runtime engine (Mule) flows and components in your application network. For applications and API gateways deployed to Anypoint Runtime Fabric (Runtime Fabric), Anypoint Monitoring provides basic metrics to all customers. Titanium-plan customers have access to additional capabilities, including application log retrieval and search.
+xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule flows and components in your application network. Anypoint Monitoring runs in a sidecar in pod deployments and gathers basic metrics for applications and API gateways deployed to Runtime Fabric. Titanium-plan customers have access to additional capabilities, including application log retrieval and search.
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/manage-monitor-applications.adoc]: "as a sidecar" instead of "in a sidecar"

Comment (jeff-1amstudios) [reply modules/ROOT/pages/manage-monitor-applications.adoc]: "in application deployments"

File: modules/ROOT/pages/index-self-managed.adoc [hunk 5]
```diff
@@ -108,6 +108,14 @@ Runtime Fabric on Self-Managed Kubernetes does not include external log forwardi
 
 For Titanium customers, Runtime Fabric on Self-Managed Kubernetes supports logging using Anypoint Monitoring. See xref:monitoring::logs.adoc[Logs in Anypoint Monitoring] for more information.
 
+=== Monitoring
+
+xref:monitoring::index.adoc[Anypoint Monitoring] provides metrics for applications and API gateways deployed to Runtime Fabric. 
+
+To collect metrics, Anypoint Monitoring sidecars run in all applications deployed to Runtime Fabric. See xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric] for details, including how to enable or disable monitoring. 
+
+Runtime Fabric does not support third-party monitoring solutions.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/index-self-managed.adoc]:
```suggestion
Runtime Fabric does not provide support for integrating third-party monitoring solutions.
```

File: modules/ROOT/pages/manage-monitor-applications.adoc (hunk 4 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/manage-monitor-applications.adoc]:
```suggestion
xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule flows and components in your application network. Anypoint Monitoring runs as a sidecar in application deployments and gathers metrics for applications and API gateways deployed to Runtime Fabric. Titanium-plan customers have access to additional capabilities, including application log retrieval and search.
```

File: modules/ROOT/pages/index-self-managed.adoc (hunk 3 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/index-self-managed.adoc]: I like that suggestion

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 6]
```diff
@@ -3,15 +3,15 @@
 Applications and API gateways deployed to Runtime Fabric include native support for Anypoint Monitoring to allow viewing metrics within Anypoint Platform.
 
 [NOTE]
-Monitoring solutions other than Anypoint Monitoring are not supported.
+Runtime Fabric does not support monitoring solutions other than Anypoint Monitoring.
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/manage-monitor-applications.adoc]: same wording as above - "provide support"?

Comment (vli63127) [reply modules/ROOT/pages/manage-monitor-applications.adoc]: yes i like the provide support

Comment (IsaacEldridge) [inline modules/ROOT/pages/manage-monitor-applications.adoc]:
```suggestion
Runtime Fabric does not provide support for integrating third-party monitoring solutions.
```

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**